CTFd/teams.py

```diff
@@ -197,6 +197,16 @@ def new():
             description="Team creation is currently disabled. Please join an existing team.",
         )
 
+    num_teams_limit = int(get_config("num_teams", default=0))
+    num_teams = Teams.query.filter_by(banned=False, hidden=False).count()
+    if 0 < num_teams_limit <= num_teams:
```

Comment:
Same simplification here right? 

```suggestion
    if num_teams_limit and num_teams >= num_teams_limit:
```